airflow/models/xcom.py

```diff
@@ -408,8 +408,9 @@ def clear(
         execution_date: Optional[pendulum.DateTime] = None,
         dag_id: Optional[str] = None,
         task_id: Optional[str] = None,
-        run_id: Optional[str] = None,
         session: Session = NEW_SESSION,
+        *,
+        run_id: Optional[str] = None,
```

Comment:
Why run_id as keyword only?

Reply:
It's a _new_ part of the API for AIP-39 Timetables  -- this used to take `execution_date`, and the only way of telling them apart is to make run_id  keyword only.

Reply:
Thought so.